### ISSUE TITLE
The 3rd "Experimental" Theme (Now Standard Worldwide)

### ISSUE THREAD

**Issue by KingGamingYT** [omnidevZero/YouTubeRedux#64]
Infected again, and this is even worse, ick
![image](https://user-images.githubusercontent.com/45918062/189801042-7a888e6c-816e-421a-a07d-100fe61aeafe.png)

**Comment by KingGamingYT** [comment]
![image](https://user-images.githubusercontent.com/45918062/190023193-e4141209-de84-4a63-970a-52386215607f.png)
![image](https://user-images.githubusercontent.com/45918062/190023158-662b8cfb-77ae-4fc5-8aa7-667817e45bde.png)
gonna be posting more as I find more.

**Comment by KingGamingYT** [comment]
Making a new comment for this, the new sneak peek update from youtube shows off the upcoming new playlist layout along with other things, this is a doozy.
https://www.youtube.com/watch?v=czIyqEC4V-s

**Comment by Thespikedballofdoom** [comment]
We need this addon now more than ever, I really hope a fix is on the way. I don't want this to be the thing that cracks me and forces me to become a programmer!

**Comment by gameshowrecaps** [comment]
Yeah we need an update badly. It's weird liking videos under the save button.

**Comment by Franck-Nein** [comment]
@KingGamingYT Some of those issues might be caused by my fix from https://github.com/omnidevZero/YouTubeRedux/issues/73#issuecomment-1278576872 could you try disabling it? If that solves the issue, please try this version, just in case [quickYoutubeLikesFix.user.css](https://gist.githubusercontent.com/Franck-Nein/905d3da23702cf58f7752029b25861f6/raw/quickYoutubeLikesFix.user.css) ?

**Comment by KingGamingYT** [comment]
> @KingGamingYT Some of those issues might be caused by my fix from [#73 (comment)](https://github.com/omnidevZero/YouTubeRedux/issues/73#issuecomment-1278576872) could you try disabling it? If that solves the issue, please try this version, just in case [quickYoutubeLikesFix.user.css](https://gist.githubusercontent.com/Franck-Nein/905d3da23702cf58f7752029b25861f6/raw/quickYoutubeLikesFix.user.css) ?

Sorry for the mega-late response, but this does indeed get rid of the blue dots. Thanks.

**Comment by Thespikedballofdoom** [comment]
I've been attempting to patch a fix for each new infection strain, but I'm running into a problem where just refreshing the page seems to prompt youtube to roll the dice to choose which mutation to infect me with, even on the same account and video. I can see why this one is a difficult problem to make an update for

**Comment by TorutheRedFox** [comment]
yeah

**Comment by TorutheRedFox** [comment]
now the older theme is no longer a part of the "roulette"

**Comment by KingGamingYT** [comment]
> now the older theme is no longer a part of the "roulette"

On one singular account, I still have the old layout, but on every other account I have the newer one

**Comment by gameshowrecaps** [comment]
Every account of mine has the newer layout, sooo RIP. Hope someone has a fix very soon. I think 75%-80% of people have the layout now.

**Comment by Thespikedballofdoom** [comment]
I believe I may have found a temporary (at least?) for the experiment userscript workaround. Youtube exposes these 'experiment' flags to the user. The guys over in the 7kttube discord put together this script to affect them I've been modifying. I can't get the script to work in firemonkey but firemonkey is breaking everything I use anyway. In violentmonkey/tampermonkey on firefox, I've been able to remove the rounded description on the darker&rounder mutation.
https://gist.github.com/Thespikedballofdoom/66b6f781c164967b86915faffbcdd8f4
To see what experiments are running, use this string in the console: `yt.config_.EXPERIMENT_FLAGS`

I copy the list from H5 to the final line and save it in a text file, and did this for each mutation I was getting, then compared them. They seem to be consistent from code to visual appearance. Unfortunately I couldn't get the experiment lists from all of them.
![DarkerRounderModified](https://user-images.githubusercontent.com/43627353/198155450-6835501f-b32d-4786-b61b-21fa9b8a2b2f.png)
![DarkerRounderNormal](https://user-images.githubusercontent.com/43627353/198155452-d0d14c84-1545-4bc0-9098-7fb373929b4b.png)
[DarkerRounderNormal.txt](https://github.com/omnidevZero/YouTubeRedux/files/9874377/DarkerRounderNormal.txt)
![Standard](https://user-images.githubusercontent.com/43627353/198155455-17ef1ffc-dfea-49a6-888c-48becfff0e7e.png)
![StandardBubbly](https://user-images.githubusercontent.com/43627353/198155457-346abfe3-ddf3-4520-83ab-ed1c6d68f2dc.png)
[StandardBubbly.txt](https://github.com/omnidevZero/YouTubeRedux/files/9874378/StandardBubbly.txt)
[StandardBubblyTry2.txt](https://github.com/omnidevZero/YouTubeRedux/files/9874379/StandardBubblyTry2.txt)
![StandardDualCards](https://user-images.githubusercontent.com/43627353/198155458-9db4b7eb-5392-47a9-b055-fb9997586720.png)
[StandardDualCards.txt](https://github.com/omnidevZero/YouTubeRedux/files/9874380/StandardDualCards.txt)

I hope this all helps.

**Comment by KingGamingYT** [comment]
This is progress, but with Redux enabled, at least for me, it already overrides rounded descriptions.

**Comment by Thespikedballofdoom** [comment]
![image](https://user-images.githubusercontent.com/43627353/198191299-036720dd-36a3-40b7-b6d0-10e7aaea70ba.png)
I'll upload a pull request as soon as I can be sure this works for a release.

**Comment by Thespikedballofdoom** [comment]
I've fixed the placement of likes buttons in the "DarkerRounder" mutation, and made sure it works in between page loads too. I'll submit a pull request as soon as I can get the styling down.
![image](https://user-images.githubusercontent.com/43627353/198205533-3386f376-48cd-4b0b-8f1e-1884b9528c14.png)

**Comment by KingGamingYT** [comment]
> I've fixed the placement of likes buttons in the "DarkerRounder" mutation, and made sure it works in between page loads too. Any other mutation loads the same way they used to, so they're fully compatible *Least I'm pretty sure*. I'll submit a pull request as soon as I can get the styling down. ![image](https://user-images.githubusercontent.com/43627353/198205533-3386f376-48cd-4b0b-8f1e-1884b9528c14.png)

Exciting.

**Comment by TorutheRedFox** [comment]
> I believe I may have found a temporary (at least?) for the experiment userscript workaround. Youtube exposes these 'experiment' flags to the user. The guys over in the 7kttube discord put together this script to affect them I've been modifying. I can't get the script to work in firemonkey but firemonkey is breaking everything I use anyway. In violentmonkey/tampermonkey on firefox, I've been able to remove the rounded description on the darker&rounder mutation. https://gist.github.com/Thespikedballofdoom/66b6f781c164967b86915faffbcdd8f4 To see what experiments are running, use this string in the console: `yt.config_.EXPERIMENT_FLAGS`
> 
> I copy the list from H5 to the final line and save it in a text file, and did this for each mutation I was getting, then compared them. They seem to be consistent from code to visual appearance. Unfortunately I couldn't get the experiment lists from all of them. These are just my names for them, youtube seems to call darkerrounder "watch_metadata_refresh".
> 
> DarkerRounderNormal ![DarkerRounderNormal](https://user-images.githubusercontent.com/43627353/198155452-d0d14c84-1545-4bc0-9098-7fb373929b4b.png) DarkerRounderModified ![DarkerRounderModified](https://user-images.githubusercontent.com/43627353/198155450-6835501f-b32d-4786-b61b-21fa9b8a2b2f.png) [DarkerRounderNormal.txt](https://github.com/omnidevZero/YouTubeRedux/files/9874377/DarkerRounderNormal.txt) Standard ![Standard](https://user-images.githubusercontent.com/43627353/198155455-17ef1ffc-dfea-49a6-888c-48becfff0e7e.png) StandardBubbly ![StandardBubbly](https://user-images.githubusercontent.com/43627353/198155457-346abfe3-ddf3-4520-83ab-ed1c6d68f2dc.png) [StandardBubbly.txt](https://github.com/omnidevZero/YouTubeRedux/files/9874378/StandardBubbly.txt) [StandardBubblyTry2.txt](https://github.com/omnidevZero/YouTubeRedux/files/9874379/StandardBubblyTry2.txt) StandardDualCards ![StandardDualCards](https://user-images.githubusercontent.com/43627353/198155458-9db4b7eb-5392-47a9-b055-fb9997586720.png) [StandardDualCards.txt](https://github.com/omnidevZero/YouTubeRedux/files/9874380/StandardDualCards.txt)
> 
> I hope this all helps.

be sure to disable yt.config_.EXPERIMENT_FLAGS.kevlar_watch_cinematics too

**Comment by Thespikedballofdoom** [comment]
Still don't know what that one does

**Comment by TorutheRedFox** [comment]
it enables the ambient lighting behind the player

**Comment by KingGamingYT** [comment]
I still question what good that feature does for the actual viewing experience, the player is a flat shape

**Comment by Thespikedballofdoom** [comment]
I've submitted my PR for firefox, gonna add it into chrome next

**Comment by gameshowrecaps** [comment]
Just saw it. It actually looks good. Now we wait for it to release to public.

**Comment by baritzu** [comment]
> > I believe I may have found a temporary (at least?) for the experiment userscript workaround. Youtube exposes these 'experiment' flags to the user. The guys over in the 7kttube discord put together this script to affect them I've been modifying. I can't get the script to work in firemonkey but firemonkey is breaking everything I use anyway. In violentmonkey/tampermonkey on firefox, I've been able to remove the rounded description on the darker&rounder mutation. https://gist.github.com/Thespikedballofdoom/66b6f781c164967b86915faffbcdd8f4 To see what experiments are running, use this string in the console: `yt.config_.EXPERIMENT_FLAGS`
> > I copy the list from H5 to the final line and save it in a text file, and did this for each mutation I was getting, then compared them. They seem to be consistent from code to visual appearance. Unfortunately I couldn't get the experiment lists from all of them. These are just my names for them, youtube seems to call darkerrounder "watch_metadata_refresh".
> > DarkerRounderNormal ![DarkerRounderNormal](https://user-images.githubusercontent.com/43627353/198155452-d0d14c84-1545-4bc0-9098-7fb373929b4b.png) DarkerRounderModified ![DarkerRounderModified](https://user-images.githubusercontent.com/43627353/198155450-6835501f-b32d-4786-b61b-21fa9b8a2b2f.png) [DarkerRounderNormal.txt](https://github.com/omnidevZero/YouTubeRedux/files/9874377/DarkerRounderNormal.txt) Standard ![Standard](https://user-images.githubusercontent.com/43627353/198155455-17ef1ffc-dfea-49a6-888c-48becfff0e7e.png) StandardBubbly ![StandardBubbly](https://user-images.githubusercontent.com/43627353/198155457-346abfe3-ddf3-4520-83ab-ed1c6d68f2dc.png) [StandardBubbly.txt](https://github.com/omnidevZero/YouTubeRedux/files/9874378/StandardBubbly.txt) [StandardBubblyTry2.txt](https://github.com/omnidevZero/YouTubeRedux/files/9874379/StandardBubblyTry2.txt) StandardDualCards ![StandardDualCards](https://user-images.githubusercontent.com/43627353/198155458-9db4b7eb-5392-47a9-b055-fb9997586720.png) [StandardDualCards.txt](https://github.com/omnidevZero/YouTubeRedux/files/9874380/StandardDualCards.txt)
> > I hope this all helps.
> 
> be sure to disable yt.config_.EXPERIMENT_FLAGS.kevlar_watch_cinematics too

![image](https://user-images.githubusercontent.com/67552711/198856849-5e59977b-eed3-44af-933d-e82ed0385e7f.png)

this script isn't perfect, notice the black bars. perhaps it was made only for dark mode?

**Comment by Thespikedballofdoom** [comment]
I don't know whats causing that but there was a better script I didn't know about at the time I'm using now, I added more flags as well https://gist.github.com/Thespikedballofdoom/122c7cd62200042f9e662662f31b9544

**Comment by Thespikedballofdoom** [comment]
It seems like the new UI has been worldwided.

**Comment by KingGamingYT** [comment]
The mobile app has been updated for nearly everyone, for the most part, so yep, seems it's been fully rolled out.

**Comment by KingGamingYT** [comment]
> I don't know whats causing that but there was a better script I didn't know about at the time I'm using now, I added more flags as well https://gist.github.com/Thespikedballofdoom/122c7cd62200042f9e662662f31b9544

Also this script completely flew over my head but yeah that fixes nearly all page layout issues. amazing work

**Comment by KingGamingYT** [comment]
![image](https://user-images.githubusercontent.com/45918062/199162362-f98351d3-22e9-45ec-bc5b-624d6fc41ec6.png)
Odd but small bug, specifically the videos tab and only the videos tab formats incorrectly on channels.

**Comment by TorutheRedFox** [comment]
here's some fixes i personally use

```CSS
/* fixes for new layout */ 
 ytd-segmented-like-dislike-button-renderer {
   float: right;
 }
 #menu-container {
   margin-bottom: 42px;
 }

 html[dark] ytd-masthead, html[dark]
#guide-content {
   background: #222222;
 }

 ytd-merch-shelf-renderer {
   display: none;
 }

.ytp-ce-video.ytp-ce-medium-round, .ytp-ce-playlist.ytp-ce-medium-round, .ytp-ce-medium-round .ytp-ce-expanding-overlay-background, .ytp-videowall-still-round-medium .ytp-videowall-still-image, .ytp-videowall-still-round-large .ytp-videowall-still-image, .ytp-ce-video.ytp-ce-large-round, .ytp-ce-playlist.ytp-ce-large-round, .ytp-ce-large-round .ytp-ce-expanding-overlay-background {
    border-radius: 0px;
}

.ytp-settings-menu.ytp-rounded-menu, .ytp-screen-mode-menu.ytp-rounded-menu, .ytp-sb-subscribe.ytp-sb-rounded, .ytp-sb-unsubscribe.ytp-sb-rounded {
  border-radius: 2px;
}

#reduxSubDiv {
    margin-top: -0px !important;
}

#avatar, #avatar > #img {
  width: 48px !important;
  height: 48px;
}

#top-row.ytd-video-secondary-info-renderer {
    margin-bottom: 6px;
    padding-top: 8px;
}

.ytd-video-owner-renderer ytd-badge-supported-renderer {
  padding-bottom: 8px;
}

span.ytd-video-view-count-renderer {
  font-weight: normal;
  font-size: 19px;
}

yt-formatted-string[is-empty]:not(.use-shadow){display: unset !important;}
 #related #metadata-line.ytd-video-meta-block span:nth-child(2) {display: unset !important;}
 #secondary-text {display: none !important;}
 #related #metadata-line.ytd-video-meta-block > .ytd-video-meta-block:not(:last-of-type)::after {display: unset !important;}

 #metadata-line.ytd-video-meta-block>.ytd-video-meta-block:not(:first-of-type) {
   display: none;
 }
```

**Comment by TorutheRedFox** [comment]
only issue left on watch page with it is the fact that return youtube dislike is having trouble finding the dislike button and spamming the log with errors, other pages still have issues though

![image](https://user-images.githubusercontent.com/15708278/199271626-95384749-2296-40e0-8e6b-ed57ee1eca75.png)

**Comment by KingGamingYT** [comment]
> only issue left on watch page with it is the fact that return youtube dislike is having trouble finding the dislike button and spamming the log with errors, other pages still have issues though
> 
> ![image](https://user-images.githubusercontent.com/15708278/199271626-95384749-2296-40e0-8e6b-ed57ee1eca75.png)

yeah due to RYTD’s broken-ness there’s just constant console spam, I’d recommend just disabling the extension or userscript until they get it fixed, the likes of which seems to be slow right now

**Comment by TorutheRedFox** [comment]
yeah

**Comment by KingGamingYT** [comment]
> here's some fixes i personally use
> 
> ```css
> /* fixes for new layout */ 
>  ytd-segmented-like-dislike-button-renderer {
>    float: right;
>  }
>  #menu-container {
>    margin-bottom: 42px;
>  }
> 
>  html[dark] ytd-masthead, html[dark]
> #guide-content {
>    background: #222222;
>  }
> 
>  ytd-merch-shelf-renderer {
>    display: none;
>  }
> 
> .ytp-ce-video.ytp-ce-medium-round, .ytp-ce-playlist.ytp-ce-medium-round, .ytp-ce-medium-round .ytp-ce-expanding-overlay-background, .ytp-videowall-still-round-medium .ytp-videowall-still-image, .ytp-videowall-still-round-large .ytp-videowall-still-image, .ytp-ce-video.ytp-ce-large-round, .ytp-ce-playlist.ytp-ce-large-round, .ytp-ce-large-round .ytp-ce-expanding-overlay-background {
>     border-radius: 0px;
> }
> 
> .ytp-settings-menu.ytp-rounded-menu, .ytp-screen-mode-menu.ytp-rounded-menu, .ytp-sb-subscribe.ytp-sb-rounded, .ytp-sb-unsubscribe.ytp-sb-rounded {
>   border-radius: 2px;
> }
> 
> #reduxSubDiv {
>     margin-top: -0px !important;
> }
> 
> #avatar, #avatar > #img {
>   width: 48px !important;
>   height: 48px;
> }
> 
> #top-row.ytd-video-secondary-info-renderer {
>     margin-bottom: 6px;
>     padding-top: 8px;
> }
> 
> .ytd-video-owner-renderer ytd-badge-supported-renderer {
>   padding-bottom: 8px;
> }
> 
> span.ytd-video-view-count-renderer {
>   font-weight: normal;
>   font-size: 19px;
> }
> 
> yt-formatted-string[is-empty]:not(.use-shadow){display: unset !important;}
>  #related #metadata-line.ytd-video-meta-block span:nth-child(2) {display: unset !important;}
>  #secondary-text {display: none !important;}
>  #related #metadata-line.ytd-video-meta-block > .ytd-video-meta-block:not(:last-of-type)::after {display: unset !important;}
> 
>  #metadata-line.ytd-video-meta-block>.ytd-video-meta-block:not(:first-of-type) {
>    display: none;
>  }
> ```

It's probably just conflicts with my existing code but putting this in does a few weird things to the watch page, like dispositioning some things on the right side
![image](https://user-images.githubusercontent.com/45918062/199282675-534b05ad-8cab-4c96-af91-dc7466249e97.png)
and getting rid of relative upload times
![image](https://user-images.githubusercontent.com/45918062/199282778-4762cebc-c02c-4485-8843-d09bad114836.png)
(looks like this with it off)
![image](https://user-images.githubusercontent.com/45918062/199283081-c0aa27ee-29e5-4ff7-8de9-aa4beee74247.png)

**Comment by TorutheRedFox** [comment]
it gets rid of upload times because they're broken for me, and it's an excerpt of a larger collection of custom css I have,

here's the whole thing:

```CSS
/* player button size css patches */

.ytp-chrome-controls .ytp-button[aria-pressed=true]:after {
    width: 20px;
    left: 8px;
}

.ytp-chrome-controls .ytp-button[aria-pressed]:after {
  bottom: 5px;
}

.ytp-big-mode .ytp-chrome-bottom {
    height: 54px;
    padding-top: 4px;
}

.ytp-big-mode .ytp-progress-bar-container {
    bottom: 52.5px;
    height: 8px;
}

.ytp-big-mode .ytp-chrome-controls {
    height: 54px;
    line-height: 54px;
}

.ytp-big-mode .ytp-time-display {
    line-height: 53px;
}

.ytp-big-mode .ytp-chrome-top .ytp-button, .ytp-big-mode .ytp-chrome-controls .ytp-button {
    width: 54px;
}

.ytp-big-mode .ytp-chrome-controls .ytp-play-button {
    width: 69px;
}

.ytp-chrome-bottom {
    height: 36px;
}

.ytp-progress-bar-container {
    bottom: 35px;
    height: 5px;
}

.ytp-chrome-controls {
    height: 36px;
    line-height: 36px;
}

.ytp-small-mode .ytp-time-display, .ytp-time-display {
    line-height: 36px;
}

.ytp-chrome-top .ytp-button, .ytp-chrome-controls .ytp-button, .ytp-replay-button {
    opacity: .9;
    width: 36px;
    padding: 0px;
}

.ytp-chrome-controls .ytp-button.ytp-mute-button {
  padding: 3.1px;
}

.ytp-chrome-controls .ytp-play-button {
    width: 46px;
}

.ytp-chrome-controls .ytp-button {
    height: 100%;
}

.ytp-chapter-title.ytp-button, .ytp-big-mode .ytp-chapter-title.ytp-button, .ytp-small-mode .ytp-chrome-controls .ytp-chapter-title.ytp-button, .ytp-embed-mobile-exp .ytp-chrome-controls .ytp-chapter-title.ytp-button {
    width: 100%;
    display: -webkit-box;
    display: -webkit-flex;
    display: -moz-box;
    display: -ms-flexbox;
    display: flex;
    -webkit-box-align: center;
    -webkit-align-items: center;
    -moz-box-align: center;
    -ms-flex-align: center;
    align-items: center;
}

.ytp-chrome-top .ytp-cards-button {
  margin-top: 7px;
}

/* youtube redux css patches */

ytd-expander#expander {
  overflow: hidden;
}

span.ytd-video-owner-renderer {
  line-height: 24px; 
  height: 22px;
  padding: 0 6px;
  font-size: 11px;
  text-align: center;
}

yt-formatted-string.style-scope.ytd-channel-name::before {
  content: "by ";
}

.style-scope.ytd-search .style-scope.ytd-channel-name::before {
    color: var(--yt-endpoint-color,var(--yt-spec-text-primary));
}

.ytd-video-secondary-info-renderer yt-formatted-string.style-scope.ytd-channel-name::before, .channel-link yt-formatted-string.style-scope.ytd-channel-name::before, .ytd-author-comment-badge-renderer yt-formatted-string.style-scope.ytd-channel-name::before, .ytd-c4-tabbed-header-renderer yt-formatted-string.style-scope.ytd-channel-name::before {
  content: "";
}

#container.ytd-masthead {
    height: 49px;
}

ytd-watch-flexy:not([theater]):not([fullscreen]) #primary.ytd-watch-flexy, ytd-watch-flexy:not([theater]):not([fullscreen]) #secondary.ytd-watch-flexy {
    padding-top: 4px !important;
}

#guide-spacer.ytd-app {
    margin-top: 50px;
}

ytd-feed-filter-chip-bar-renderer {
    margin-left: 0px;
}

#chips-wrapper.ytd-feed-filter-chip-bar-renderer {
    width: var(--ytd-rich-grid-chips-bar-width);
}



ytd-menu-renderer ytd-menu-renderer .ytd-menu-renderer[style-target=button], .ytd-menu-renderer[style-target=button], ytd-button-renderer[is-icon-button][style-action-button],
ytd-download-button-renderer  {
    vertical-align: top;
}

ytd-download-button-renderer {
  padding-left: 6px;
}

ytd-video-primary-info-renderer:after {
    --yt-icon-button-icon-width: 24px;
    --yt-icon-button-icon-height: 24px;
    width: var(--yt-icon-width);
    height: 32px;
    content: "";

    display: inline-block;
    box-sizing: border-box;
    font-size: 0;
}

#button.dropdown-trigger {
  height: 36px;
}

yt-formatted-string.style-scope.ytd-toggle-button-renderer.style-text, yt-formatted-string.style-scope.ytd-toggle-button-renderer.style-default-active {
  font-weight: 500;
  font-size: 11px;
  text-decoration: none;
  white-space: nowrap;
  word-wrap: normal;
  line-height: normal;
  vertical-align: middle;
}

ytd-video-primary-info-renderer {
    padding: 15px;
    padding-bottom: 5px;
    position: relative;
}

#info.ytd-watch-flexy {
  min-height: 150px;
}

.redux-moved-info:last-of-type + yt-icon-button {
    transform: translateY(-2px);
}
yt-formatted-string.style-scope.ytd-subscribe-button-renderer {
  font-weight: normal;
  font-size: 100%;
}

/* small thumbnails */
ytd-compact-video-renderer:not([watch-feed-big-thumbs]) ytd-thumbnail.ytd-compact-video-renderer, ytd-compact-radio-renderer:not([watch-feed-big-thumbs]) ytd-thumbnail.ytd-compact-radio-renderer, ytd-compact-playlist-renderer:not([watch-feed-big-thumbs]) ytd-playlist-thumbnail.ytd-compact-playlist-renderer, ytd-compact-movie-renderer:not([watch-feed-big-thumbs]) ytd-thumbnail.ytd-compact-movie-renderer {
    height: 67px;
    width: 120px;
}

ytd-playlist-renderer[is-search] ytd-playlist-thumbnail.ytd-playlist-renderer, 
#avatar-section.ytd-channel-renderer {
    max-width: 246px;
}

ytd-button-renderer.ytd-item-section-renderer, ytd-compact-radio-renderer.ytd-item-section-renderer, ytd-compact-video-renderer.ytd-item-section-renderer, ytd-compact-movie-renderer.ytd-item-section-renderer, ytd-compact-playlist-renderer.ytd-item-section-renderer, ytd-emergency-onebox-renderer.ytd-item-section-renderer, ytd-feed-nudge-renderer.ytd-item-section-renderer, ytd-movie-renderer.ytd-item-section-renderer, ytd-playlist-renderer.ytd-item-section-renderer, ytd-show-renderer.ytd-item-section-renderer, ytd-radio-renderer.ytd-item-section-renderer, yt-search-no-results-renderer.ytd-item-section-renderer, ytd-video-renderer.ytd-item-section-renderer, ytd-channel-renderer.ytd-item-section-renderer, ytd-promoted-sparkles-text-search-renderer.ytd-item-section-renderer, ytd-web-answer-renderer.ytd-item-section-renderer, ytd-clarification-renderer.ytd-item-section-renderer, ytd-fact-check-renderer.ytd-item-section-renderer, ytd-info-panel-container-renderer.ytd-item-section-renderer {
    margin-top: 15px;
}

#secondary-inner.ytd-watch-flexy #related, #secondary-inner.ytd-watch-flexy #playlist + #related:not(#secondary-inner.ytd-watch-flexy #playlist[hidden="true"] + #related) {
  padding-top: 0px;
}

.ytd-compact-video-renderer .badge-style-type-simple, .ytd-compact-video-renderer .badge-style-type-live-now-alternate {
    margin-top: -18px;
    font-size: 11px;
    height: 10px;
    padding-top: 1px;
    position: absolute;
    left: 53%;
}

ytd-rich-item-renderer .badge-style-type-simple, .ytd-rich-item-renderer .badge-style-type-live-now-alternate {
    margin-top: -28px;
    position: absolute;
    right: 0;
    font-size: 11px;
    height: 14px;
    padding-top: 1px;
}

#meta.ytd-rich-grid-media {
    overflow-y: hidden;
}

.ytd-compact-video-renderer .badge-style-type-live-now-alternate {
  padding-top: 1px;
  padding-bottom: 4px;
}

div #show-more-related {
  padding-top: 11px;
  padding-bottom: 8px;
}

#related {
  margin-bottom: 32px;
}

#commentbox #creation-box, #placeholder-area.ytd-comment-simplebox-renderer {
    min-height: 32px;
}

#action-buttons.ytd-comment-renderer {
    margin-top: -2px;
}

#more-replies, #less-replies {
  margin-top: 2px;
  margin-bottom: 0px;
}


.ytd-comment-thread-renderer {
  margin-bottom: -16px;
  margin-top: 8px;
}

ytd-comments-header-renderer {
    margin-top: 12px;
    margin-bottom: 12px;
}

#title.ytd-comments-header-renderer {
    margin-bottom: 12px;
}

#meta.ytd-watch-flexy tp-yt-paper-button#more {
  margin-top: 0px;
}

ytd-video-secondary-info-renderer {
    padding-bottom: 4px !important;
}

#show-more-related {
  padding-bottom: 0px;
}

.style-scope.ytd-comment-replies-renderer #commentbox #creation-box::before {
  top: -4px;
} 

#count.ytd-comments-header-renderer yt-formatted-string span:last-child::after {
    content: '(';
    margin: 0 5px;
    margin-right: 0px;
    min-width: 160px;
}

ytd-video-view-count-rende

#count.ytd-comments-header-renderer yt-formatted-string span:last-child::before {
    content: ' All';
    margin: 0 0;
}

#count.ytd-comments-header-renderer yt-formatted-string span:first-child::after {
    content: ')';
}

.ytp-spinner {
  transform: scale(0.5) translateZ(0);
}

#logo.ytd-topbar-logo-renderer, ytd-yoodle-renderer.ytd-topbar-logo-renderer {
  margin-top: 0px;
}

#author-comment-badge.ytd-comment-renderer, #sponsor-comment-badge.ytd-comment-renderer, #header.ytd-comment-renderer {
  margin-top: -2px;
  margin-bottom: 0px;
}

#pinned-comment-badge.ytd-comment-renderer {
    margin-bottom: 0px;
}

.ytd-comment-renderer #text.ytd-channel-name {
    font-size: var(--ytd-channel-name-text-font-size);
}

.ytd-comment-renderer[creator] {
  background-color: #0465f1;
  color: #fff;
}

ytd-author-comment-badge-renderer[creator] {
  height: 16px;
  border-radius: 0px;
  margin-bottom: 0px;
  padding-right: 2px;
  padding-left: 2px;
}

#content-text.ytd-comment-renderer {
  font-size: 14px;
}

.bold.yt-formatted-string {
    font-weight: bold;
}

#content-text.ytd-comment-renderer {
    line-height: 16px !important;
}

#author-thumbnail.ytd-comment-renderer yt-img-shadow.ytd-comment-renderer {
    margin-right: 12px;
}

ytd-comment-replies-renderer.ytd-comment-thread-renderer {
  margin-left: 52px;
}

.ytd-comment-replies-renderer #button.ytd-button-renderer #text.ytd-button-renderer span:first-child::after {
  content: "all ";
}

.ytd-comment-replies-renderer #button.ytd-button-renderer #text.ytd-button-renderer span:first-child::after {
  content: "all ";
}

.ytd-comment-replies-renderer #button.ytd-button-renderer yt-icon.ytd-button-renderer+yt-formatted-string.ytd-button-renderer, #expander.ytd-comment-replies-renderer .expander-header.ytd-comment-replies-renderer[teaser] #more-replies.ytd-comment-replies-renderer {
  order: 0;
  margin-left: -4px;
  font-size: 14px;
}

#expander.ytd-comment-replies-renderer .expander-header.ytd-comment-replies-renderer[teaser] #more-replies.ytd-comment-replies-renderer {
  margin-left: 2px;
}

ytd-comment-replies-renderer:not([modern]) #expander .ytd-comment-replies-renderer .dot.ytd-comment-replies-renderer
{
  display: none;
}
.ytd-comment-replies-renderer ytd-button-renderer[is-paper-button] yt-icon.ytd-button-renderer, ytd-comment-replies-renderer:not([modern]) .more-button.ytd-comment-replies-renderer {
  order: 1;
  margin-left: 2px;
}

 ytd-comment-replies-renderer:not([modern]) .more-button.ytd-comment-replies-renderer {
  height: 20px;
  padding-bottom: 4px;
  margin-top: 4px;
 }

.ytd-comment-replies-renderer #button {
  --yt-button-padding: 0px 0 8px 0;
}

.ytd-comment-replies-renderer .ytd-comment-action-buttons-renderer #button {
  --yt-button-padding: 0px 0 0px 0;
}

/* set font family for everything */
* { 
  font-family: Arial, Helvetica, sans-serif;
}

#playlist {
  padding-left: 12px;
}

#upload-info.ytd-video-owner-renderer {
  margin-top: -4px;
}

#search-form.ytd-searchbox, #search-icon-legacy.ytd-searchbox {
  height: 29px;
}

#search-icon-legacy.ytd-searchbox yt-icon.ytd-searchbox {
  width: 18px;
  height: 18px;
  margin-top: -1px;
  margin-left: -2px;
}

#country-code.ytd-topbar-logo-renderer {
  margin-top: 17px;
}

#container.ytd-searchbox {
  margin-left: -10px;
}

ytd-searchbox.ytd-masthead {
  margin-right: 29px;
}

[dark] #search-icon-legacy.ytd-searchbox {
  border-color: #404040;
}

#search-icon-legacy.ytd-searchbox {
  border-color: #d3d3d3;
  border-left: 0;
  -moz-border-radius-topleft: 0;
  border-top-left-radius: 0;
  -moz-border-radius-bottomleft: 0;
  border-bottom-left-radius: 0;
  width: 66px;
}

#search-icon.ytd-searchbox {
  display: none;
  width: 0px;
}

ytd-searchbox[has-focus] #container.ytd-searchbox {
  margin-left: -10px !important;
  padding: 2px 6px;
}

ytd-video-renderer[is-search] ytd-thumbnail.ytd-video-renderer {
  height: 137px;
}

#primary.ytd-watch-flexy {
  padding-left: 0px;
  margin-left: 0px;
}

#primary.ytd-watch-flexy > #primary-inner {
  padding-left: 0px;
}

ytd-watch-flexy[flexy] #secondary.ytd-watch-flexy {
  padding-right: 0px;
  margin-right: 0px;
}

.title.ytd-video-primary-info-renderer, .title.style-scope.ytd-video-primary-info-renderer yt-formatted-string.ytd-video-primary-info-renderer {
  font-size: 20px;
  font-weight: normal;
  font-family: Arial, Helvetica, sans-serif;
  line-height: normal;
  word-wrap: break-word;
}

.redux-moved-info:last-of-type + yt-icon-button {
    transform: none;
}

body {
    word-wrap: break-word;
    margin: 0;
    padding: 0;
    font: 12px Arial, Helvetica, sans-serif;
}

#info.ytd-video-primary-info-renderer > #menu-container {
    transform: translateY(42px) !important;
    min-width: 160px;
}

#avatar.ytd-video-owner-renderer {
  margin-right: 12px;
}

.ytd-video-owner-renderer #text.ytd-channel-name {
    display: inline-block;
    height: 22px;
    color: #333;
    max-width: 315px;
    overflow: hidden;
    text-overflow: ellipsis;
    vertical-align: top;
    font-size: 13px;
    line-height: normal;
}

.ytd-video-owner-renderer.ytd-video-owner-renderer.ytd-video-owner-renderer.ytd-video-owner-renderer #text.ytd-channel-name.ytd-channel-name.ytd-channel-name.ytd-channel-name.ytd-channel-name:hover {
  text-decoration: underline;
  -webkit-text-decoration: underline;
}

.redux-moved-info:first-of-type yt-icon-button#button {
  margin-left: -8px;
  padding-left: 0px;
  padding-right: 0px;
}

ytd-watch-flexy[flexy][is-two-columns_] #columns.ytd-watch-flexy {
    max-width: 1066px;
    min-width: 1003px;
    width: auto;
}

@media screen and (min-width: 1294px) and (min-height: 630px) {
ytd-watch-flexy[flexy][is-two-columns_] #columns.ytd-watch-flexy {
    max-width: 1280px !important;
}
}

#items.ytd-watch-next-secondary-results-renderer {
  padding-right: 15px;
}

#primary.ytd-watch-flexy, #secondary.ytd-watch-flexy {
  margin-bottom: 10px;
}

ytd-watch-flexy:not([theater]):not([fullscreen]) #primary.ytd-watch-flexy, ytd-watch-flexy:not([theater]):not([fullscreen]) #secondary.ytd-watch-flexy {
    padding-top: 10px !important;
}

#reply-button-end {
  order:1;
}

#share-button {
  order:3;
}

#creator-heart {
  order:2;
  margin-left: -8px;
  margin-right: -8px;
}

#vote-count-middle.ytd-comment-action-buttons-renderer {
  margin-right: 8px;
}

#vote-count-left.ytd-comment-action-buttons-renderer[hidden]+#like-button.ytd-comment-action-buttons-renderer {
    margin-left: 8px;
}

#less-replies {
  margin-left: 4px;
  margin-top: 3px;
}

.badge-style-type-collection {
  display: none;
}

yt-formatted-string[is-empty]:not(.use-shadow) {
    display: unset !important;
}

/* fixes for new layout */ 
 ytd-segmented-like-dislike-button-renderer {
   float: right;
 }
 #menu-container {
   margin-bottom: 42px;
 }

 html[dark] ytd-masthead, html[dark]
#guide-content {
   background: #222222;
 }

 ytd-merch-shelf-renderer {
   display: none;
 }

.ytp-ce-video.ytp-ce-medium-round, .ytp-ce-playlist.ytp-ce-medium-round, .ytp-ce-medium-round .ytp-ce-expanding-overlay-background, .ytp-videowall-still-round-medium .ytp-videowall-still-image, .ytp-videowall-still-round-large .ytp-videowall-still-image, .ytp-ce-video.ytp-ce-large-round, .ytp-ce-playlist.ytp-ce-large-round, .ytp-ce-large-round .ytp-ce-expanding-overlay-background {
    border-radius: 0px;
}

.ytp-settings-menu.ytp-rounded-menu, .ytp-screen-mode-menu.ytp-rounded-menu, .ytp-sb-subscribe.ytp-sb-rounded, .ytp-sb-unsubscribe.ytp-sb-rounded {
  border-radius: 2px;
}

#reduxSubDiv {
    margin-top: -0px !important;
}

#avatar, #avatar > #img {
  width: 48px !important;
  height: 48px;
}

#top-row.ytd-video-secondary-info-renderer {
    margin-bottom: 6px;
    padding-top: 8px;
}

.ytd-video-owner-renderer ytd-badge-supported-renderer {
  padding-bottom: 8px;
}

span.ytd-video-view-count-renderer {
  font-weight: normal;
  font-size: 19px;
}

yt-formatted-string[is-empty]:not(.use-shadow){display: unset !important;}
 #related #metadata-line.ytd-video-meta-block span:nth-child(2) {display: unset !important;}
 #secondary-text {display: none !important;}
 #related #metadata-line.ytd-video-meta-block > .ytd-video-meta-block:not(:last-of-type)::after {display: unset !important;}

 #metadata-line.ytd-video-meta-block>.ytd-video-meta-block:not(:first-of-type) {
   display: none;
 }
```

**Comment by Narixv** [comment]
### I thought it might be helpful if I contribute my issue here as well

**As you can see, my video page is severely messed up by this new YT update : /** [](url)
![unknown](https://user-images.githubusercontent.com/103471431/199570894-4909bb08-df8f-41ea-83d2-f9ab9c9ba5c5.png)

**And these are the settings I enable on** `Redux`
![unknown1](https://user-images.githubusercontent.com/103471431/199572790-67e7c015-1018-4cfc-be08-ca95fcd41b8c.png)
![unknown2](https://user-images.githubusercontent.com/103471431/199572805-5bb3bef6-8618-4297-9630-b4a9bfcb3f60.png)

**Comment by KingGamingYT** [comment]
> ### I thought it might be helpful if I contribute my issue here as well
> **As you can see, my video page is severely messed up by this new YT update : /** ![unknown](https://user-images.githubusercontent.com/103471431/199570894-4909bb08-df8f-41ea-83d2-f9ab9c9ba5c5.png)
> 
> **And these are the settings I enable on** `Redux` ![unknown1](https://user-images.githubusercontent.com/103471431/199572790-67e7c015-1018-4cfc-be08-ca95fcd41b8c.png) ![unknown2](https://user-images.githubusercontent.com/103471431/199572805-5bb3bef6-8618-4297-9630-b4a9bfcb3f60.png)

Most of your issues will probably be remedied with Thespikedballofdoom's script (https://gist.github.com/Thespikedballofdoom/122c7cd62200042f9e662662f31b9544), you can just paste it into something like tampermonkey.

**Comment by Narixv** [comment]
### So I went ahead and added the script but its not fixing some things on the page... 🤔
@KingGamingYT https://github.com/omnidevZero/YouTubeRedux/issues/64#issuecomment-1301179672
![unknown3](https://user-images.githubusercontent.com/103471431/199598746-01ccb372-5227-47c4-9874-c80f6a232971.png)
![unknown4](https://user-images.githubusercontent.com/103471431/199598762-99c06b75-b5df-4c22-bb72-aab5700bac99.png)

**Comment by FIamey** [comment]
The script mentioned in https://github.com/omnidevZero/YouTubeRedux/issues/64#issuecomment-1301179672 does not work for me, I'm using Violent Monkey on LibreWolf. This new YouTube UI update is horrendous and I hope this extension gets fixed soon.

**Comment by Thespikedballofdoom** [comment]
https://github.com/mchangrh/yt-neuter/blob/main/userscripts/reflow.user.js
I found this in the return youtube dislike discord server. This ought to help someone and I'm using it myself now, it alters the amount of elements in grid layouts. It's compatible with redux though it looks like this: https://rdgb.net/i/KnA5T.png

> The script mentioned in [#64 (comment)](https://github.com/omnidevZero/YouTubeRedux/issues/64#issuecomment-1301179672) does not work for me, I'm using Violent Monkey on LibreWolf. This new YouTube UI update is horrendous and I hope this extension gets fixed soon.
Working for me on firefox developer, the original script writer did say it only worked on firefox and not chrome so that's unfortunate.



If omnidev doesn't come back anytime soon then I don't know what will happen, but we maybe could get a bunch of scripters together and help inform the public about everything they can still do versus the YT layout. There is a lot of stuff I've learned since joining the 7kttube discord https://discord.gg/7WRjXHMfXJ

**Comment by KingGamingYT** [comment]
> Working for me on firefox developer, the original script writer did say it only worked on firefox and not chrome so that's unfortunate.

That's odd, I'm on chrome and the script works perfectly fine, I use tampermonkey though.

> I found this in the return youtube dislike discord server. This ought to help someone and I'm using it myself now, it alters the amount of elements in grid layouts. It's sort of compatible with redux though it looks like this: https://rdgb.net/i/KnA5T.png

I'll bet that with some tweaking it can be formatted to redux properly.

**Comment by KingGamingYT** [comment]
![image](https://user-images.githubusercontent.com/45918062/199621124-a1eba53d-b693-4ec6-97fb-5499c64dbfea.png)
On a somewhat unrelated note, I've been looking over 7kttube's screenshots and toru's small thumbnail patch, and while they're similar a few tweaks need to be done before it looks right
The "New" square is overlapped by the views and time info, and view counts are shortened.
I actually made the suggestion waaaay back in February #13, and while omnidev marked it as an enhancement, it was never really given a spotlight in favor of bug fixes and more important things like trying to fix youtube's bad decisions. As such, I'm hoping to see if I can get this to look and function well enough to put in a pull request as a new feature.

**Comment by Narixv** [comment]
I quite using all chrome browsers because of the **`Manifest V3`**
> Working for me on firefox developer, the original script writer did say it only worked on firefox and not chrome so that's unfortunate.
@Thespikedballofdoom  https://github.com/omnidevZero/YouTubeRedux/issues/64#issuecomment-1301350267

So, I also switched to **`LibreWolf`** (a fork of **`Firefox`**) just like @FIamey https://github.com/omnidevZero/YouTubeRedux/issues/64#issuecomment-1301329389

Its so evil for YouTube to just push these god awful changes to us. and then to leave us to pick up the pieces from there mess.
im happy to here that there are good scriptwriters dedicated to fixing YouTube's mess ups.
> If omnidev doesn't come back anytime soon then I don't know what will happen, but we maybe could get a bunch of scripters together and help inform the public about everything they can still do versus the YT layout. There is a lot of stuff I've learned since joining the 7kttube discord https://discord.gg/7WRjXHMfXJ 
@Thespikedballofdoom https://github.com/omnidevZero/YouTubeRedux/issues/64#issuecomment-1301350267

**Comment by omnidevZero** [comment]
I'm super late for this super thread but I just got back to working on the backlog of issues with this one being the priority. Since the thread is quite hot (unlike the fix) I just pushed an update that will partially fix some of the stuff described here, with more updates coming shortly after!

**Comment by KingGamingYT** [comment]
Great to see you back, hope everything's cataloged well enough kek

**Comment by gameshowrecaps** [comment]
Thanks for the update! However it's hard for me to tell if I liked a video now with this update.

**Comment by KingGamingYT** [comment]
> Thanks for the update! However it's hard for me to tell if I liked a video now with this update.

> button[aria-pressed="true"] > .yt-spec-button-shape-next__icon > yt-icon > svg {fill: #3ea6ff !important;}
button[aria-pressed="true"] > div > span {color: #3ea6ff !important;}

@gameshowrecaps just put this into some website css themer like stylus and you should be good.

**Comment by Thespikedballofdoom** [comment]
Yay!

**Comment by Narixv** [comment]
### Yeah same here 🤔 
> Thanks for the update! However it's hard for me to tell if I liked a video now with this update.
@gameshowrecaps https://github.com/omnidevZero/YouTubeRedux/issues/64#issuecomment-1302602430

![image](https://user-images.githubusercontent.com/103471431/199835190-05292966-721d-4da9-a799-f7992da2ae22.png)

How do I add that CSS you mentioned to fix the issue gameshowrecaps had? 🤔 
> just put this into some website css themer like stylus and you should be good.
@KingGamingYT https://github.com/omnidevZero/YouTubeRedux/issues/64#issuecomment-1302608052

**Comment by Thespikedballofdoom** [comment]
> ### Yeah same here 🤔
> 
> > Thanks for the update! However it's hard for me to tell if I liked a video now with this update.
> > @gameshowrecaps [#64 (comment)](https://github.com/omnidevZero/YouTubeRedux/issues/64#issuecomment-1302602430)
> 
> ![image](https://user-images.githubusercontent.com/103471431/199835190-05292966-721d-4da9-a799-f7992da2ae22.png)
> 
> How do I add that CSS you mentioned to fix the issue gameshowrecaps had? 🤔
> 
> > just put this into some website css themer like stylus and you should be good.
> > @KingGamingYT [#64 (comment)](https://github.com/omnidevZero/YouTubeRedux/issues/64#issuecomment-1302608052)

https://add0n.com/stylus.html

**Comment by Narixv** [comment]
Thanks @Thespikedballofdoom @KingGamingYT @gameshowrecaps 

These quick fixes should be enough until @omnidevZero can make a more permanent fix inside `Redux`

Best of luck omnidevZero! 

![image](https://user-images.githubusercontent.com/103471431/199837980-ad7f85a6-a920-434a-85bb-8cdc5784bc54.png)
![image](https://user-images.githubusercontent.com/103471431/199838075-0f64ecb8-3956-4b91-8faf-f210a16fd770.png)

**Comment by omnidevZero** [comment]
Great job everybody on reporting stuff in this thread and the others! I've got a lot of catching up to do but I'll try to fix most of the critical issues asap. I wanted to have some fixes for likes/dislikes in the newest update as well but I was blocked by yet another issue in which the buttons straight up didn't work seemingly because of a YT error but it might be related to Redux moving elements around. Since broken video page layout was the most important issue I decided to push the update before fixing everything else. There are still some visual issues here and there but I'm on it! 😃

**Comment by KingGamingYT** [comment]
![image](https://user-images.githubusercontent.com/45918062/200069165-947f2e61-fd2a-48cf-a7b1-adf748fdc83c.png)
As of the latest patch dislikes have been restored for me, however spacing has become a bit messed up

**Comment by Narixv** [comment]
> As of the latest patch dislikes have been restored for me, however spacing has become a bit messed up
> 
> Edit: spacing fixed itself, and dislikes went away with it. Very odd.
@KingGamingYT https://github.com/omnidevZero/YouTubeRedux/issues/64#issuecomment-1304199583

Thats strange 🤔 my spacing looks nothing like yours.
Perhaps, its me me IDK 

![image](https://user-images.githubusercontent.com/103471431/200089235-9fc2e957-3de1-4d8e-ac26-0df539fa3f6c.png)

> Great job everybody on reporting stuff in this thread and the others! I've got a lot of catching up to do but I'll try to fix most of the critical issues asap. I wanted to have some fixes for likes/dislikes in the newest update as well but I was blocked by yet another issue in which the buttons straight up didn't work seemingly because of a YT error but it might be related to Redux moving elements around. Since broken video page layout was the most important issue I decided to push the update before fixing everything else. There are still some visual issues here and there but I'm on it! 😃
@omnidevZero https://github.com/omnidevZero/YouTubeRedux/issues/64#issuecomment-1303113595

Also, once you get all that figured out could you please add a way to remove the 'clip' and 'thanks' buttons at the bottom of the video?

**Comment by KingGamingYT** [comment]
That'll probably go along with the layout fixes, since removing the clip and thanks buttons should already be options to disable in redux's settings.

**Comment by omnidevZero** [comment]
> Also, once you get all that figured out could you please add a way to remove the 'clip' and 'thanks' buttons at the bottom of the video?

@Narixv "Clip" can already be removed via "Hide features" section of settings. Hiding the "Thanks" button will be added in the next update though!

Multiple spacing issues will be fixed in the update that will follow the upcoming one.

**Comment by KingGamingYT** [comment]
> Multiple spacing issues will be fixed in the update that will follow the upcoming one.
@omnidevZero About spacing issues, here's another one that might be somewhat hard to catch
![image](https://user-images.githubusercontent.com/45918062/200426453-fde83c23-7b82-43a1-affd-316c7a91d9d8.png)

**Comment by Thespikedballofdoom** [comment]
No 6 video channel page?
![image](https://user-images.githubusercontent.com/43627353/200466538-9a7887b8-3a20-4b94-a5df-b64d8637fb53.png)

**Comment by KingGamingYT** [comment]
I wonder if after redux has been all patched up omnidev will move on to things like further redux-ing channel pages, akin to how 7kt-Tube has done already.

**Comment by TorutheRedFox** [comment]
how does _this_ even happen?
![image](https://user-images.githubusercontent.com/15708278/200625147-32ddb2f9-97b7-4b27-9b7d-53f14c487298.png)

**Comment by KingGamingYT** [comment]
> how does _this_ even happen? ![image](https://user-images.githubusercontent.com/15708278/200625147-32ddb2f9-97b7-4b27-9b7d-53f14c487298.png)

haha what even

**Comment by Thespikedballofdoom** [comment]
Its just an unhidden old subscribed button i think

**Comment by TorutheRedFox** [comment]
> Its just an not-hidden old subscribed button i think

it's supposed to be the notification bell, but for some reason it gets the 'subscribed' text applied to it, plus it's massive lol

**Comment by Narixv** [comment]
Hello sorry for the late reply.
> 
> "Clip" can already be removed via "Hide features" section of settings. Hiding the "Thanks" button will be added in the next update though!
> 
> Multiple spacing issues will be fixed in the update that will follow the upcoming one.
@omnidevZero https://github.com/omnidevZero/YouTubeRedux/issues/64#issuecomment-1306196145

I made sure all of my `Redux` settings are correct and my other extensions are not interfering, the **"Clip"** setting is turned on and so is **"Thanks"**

![redux 1](https://user-images.githubusercontent.com/103471431/201483182-8ee25222-1ae2-4fe2-b74b-039a2c38d3e7.png)
![redux 2](https://user-images.githubusercontent.com/103471431/201483183-775e8203-5124-40b2-a9bd-f2504a483efe.png)

As you can ass in this video.

![video ex](https://user-images.githubusercontent.com/103471431/201483428-aae6781e-6156-45f8-a9b4-31c51e81be1c.png)

### In addition,

Im not sure why all of the sudden YT Thumbnails started to not display at all. This seems to have started to happen when you pushed `Ver. 2.13.6` Firefox

![YT Thumnails](https://user-images.githubusercontent.com/103471431/201483628-c03eba55-0b51-4d81-a6ea-0bd9172a2725.png)
![image](https://user-images.githubusercontent.com/103471431/201483797-3119fd35-024a-463b-ad14-fc3c461919a3.png)

**Comment by KingGamingYT** [comment]
> Hello sorry for the late reply.
> 
> > "Clip" can already be removed via "Hide features" section of settings. Hiding the "Thanks" button will be added in the next update though!

> I made sure all of my `Redux` settings are correct and my other extensions are not interfering, the **"Clip"** setting is turned on and so is **"Thanks"**

> ### In addition,
> Im not sure why all of the sudden YT Thumbnails started to not display at all. This seems to have started to happen when you pushed `Ver. 2.13.6` Firefox

I'm not sure why the buttons aren't disappearing for you. You may have separate userscripts running that interfere though.
As for the thumbnails not loading thing, I have observed the same thing on a clean firefox install running **Rehike**, so it is not a redux issue.

**Comment by Narixv** [comment]
Hello,

> 
> I'm not sure why the buttons aren't disappearing for you. You may have separate userscripts running that interfere though.
> As for the thumbnails not loading thing, I have observed the same thing on a clean firefox install running Rehike, so it is not a redux issue.
@KingGamingYT https://github.com/omnidevZero/YouTubeRedux/issues/64#issuecomment-1312590063

### As for user scripts
I do not have any running at the moment. I turned them off when @omnidevZero pushed `Redux Ver. 2.13.6`

![image](https://user-images.githubusercontent.com/103471431/201527765-2f379d75-3f0e-4e83-9c59-c617f1cce467.png)

### As for my my particular browser
its a fork of `Firefox`. I am running `LibreWolf Ver. 102.0.1-1`
However, this Thumbnail issue was not present before and I did not update my browser. 🤔 

![image](https://user-images.githubusercontent.com/103471431/201528025-a0ce7035-aa39-45ea-9006-4e3f3591a6d2.png)

**Comment by KingGamingYT** [comment]
@Narixv This may be a firefox-based client issue then.

**Comment by KingGamingYT** [comment]
A bit of an overdue title change
![image](https://user-images.githubusercontent.com/45918062/202269029-dd3307d7-1f52-49e7-b5a8-fd7ff45e4131.png)

**Comment by Thespikedballofdoom** [comment]
The universal international world's foremost authority

**Comment by omnidevZero** [comment]
![image](https://user-images.githubusercontent.com/74119439/202292362-efa36789-ed8f-42c5-bea5-208d10ddcd33.png)

**Comment by Thespikedballofdoom** [comment]
I'm getting this bug again. It happened after I turned the extension on and off in firefox, I'm guessing it's another case of install-before? I don't know what extension is fucking it.
![image](https://user-images.githubusercontent.com/43627353/203151613-74d366fb-00c5-4e14-a116-ce4c4e1028fe.png)

**Comment by omnidevZero** [comment]
Time to close this one. If there are still things that are broken, please go ahead and submit separate issues for them.
